php/class-fm-bylines-author.php

```diff
@@ -1,17 +1,35 @@
 <?php
 /**
  * Setup Author specific hooks using fieldmanager-bylines
+ *
+ * @package Fieldmanager_Bylines
  */
+
 if ( ! class_exists( 'FM_Bylines_Author' ) ) {
 
+	/**
+	 * Class FM_Bylines_Author
+	 */
 	class FM_Bylines_Author extends FM_Bylines_Post {
-
+		/**
+		 * The one true author object
+		 *
+		 * @var object
+		 */
 		private static $instance;
 
+		/**
+		 * Silence is golden
+		 */
 		private function __construct() {
```

Comment:
If you _really_ want to prevent object instantiation through any method other than `instance()` you also need to apply this pattern to `__clone()` and `__wakeup()`.